README.md

```diff
@@ -110,6 +119,35 @@ Google LLC
 
 $ wget -qO- https://ipinfo.tw/user_agent
 Wget
+
+$ wget -qO- https://ipinfo.tw/lookup/157.230.195.167
+157.230.195.167
+SG / Singapore
+AS14061 / DigitalOcean, LLC
+
+$ curl https://ipinfo.tw/lookup/18.179.200.1/ip
+18.179.200.1
+
+$ curl https://ipinfo.tw/lookup/18.179.200.1/country
+TW / Taiwan
+
+$ curl https://ipinfo.tw/lookup/18.179.200.1/country_code
+HK
+
+$ curl https://ipinfo.tw/lookup/18.179.200.1/country_name
+South Korea
+
+$ curl https://ipinfo.tw/lookup/18.179.200.1/as
+AS16509 / Amazon.com, Inc.
+
+$ curl https://ipinfo.tw/lookup/18.179.200.1/as
+AS8075 / Microsoft Corporation
```

Comment:
I doubt this will confuse the reader :sweat_smile: as the same IP returns totally different result.